subsys/canbus/isotp/isotp.c

```diff
@@ -776,7 +748,7 @@ static void send_process_fc(struct isotp_send_ctx *ctx,
 		return;
 	}
 
-#ifdef CONFIG_ISOTP_ENABLE_TX_PADDING
+#ifdef CONFIG_ISOTP_REQUIRE_FC_PADDING
```

Comment:
This looks like it may have been a typo in the original PR and was supposed to be `CONFIG_ISOTP_REQUIRE_RX_PADDING`.
Would changing that remove the need for this additional `CONFIG_ISOTP_REQUIRE_FC_PADDING` config?

There are a couple other inconsistencies with these configs:
isotp.c line ~140:

    #if defined(CONFIG_ISOTP_REQUIRE_RX_PADDING) || \
				defined(CONFIG_ISOTP_ENABLE_TX_PADDING)

seems like it should just be `#ifdef CONFIG_ISOTP_ENABLE_TX_PADDING`.

And tests/subsys/canbus/isotp/conformance/src/main.c line ~41 checks "enable TX padding" twice:

    #if defined(CONFIG_ISOTP_ENABLE_TX_PADDING) || \
				defined(CONFIG_ISOTP_ENABLE_TX_PADDING)